app/prodRelease-badging.txt

```diff
@@ -1,6 +1,6 @@
-package: name='com.google.samples.apps.nowinandroid' versionCode='8' versionName='0.1.2' platformBuildVersionName='14' platformBuildVersionCode='34' compileSdkVersion='34' compileSdkVersionCodename='14'
+package: name='com.google.samples.apps.nowinandroid' versionCode='8' versionName='0.1.2' platformBuildVersionName='15' platformBuildVersionCode='35' compileSdkVersion='35' compileSdkVersionCodename='15'
 sdkVersion:'21'
-targetSdkVersion:'34'
+targetSdkVersion:'35'
```

Comment:
Seems like this PR _does_ update to API 35. Please could you update the PR description to match.

Reply:
Looks like you need to manually update the prodReleaseBadging. You can do this by running: 

`./gradlew updateProdReleaseBadging`